plusplus.js

```diff
@@ -0,0 +1,24 @@
+const { createClient } = require('redis');
+
+const client = createClient();
+
+// Redis schema, key => value
+// username (str) => score (number as str)
+const initPlusPlus = async () => {
+
+   await client.connect();
+
+}
+
+// !++ @username
+const plusPlus = async (username) => {
+   const currentScoreStr = await client.get(username);
+   const newScore = (currentScoreStr) ? Number.parseInt(currentScoreStr) + 1 : 1;
```

Comment:
* `Number('123') = 123`
* `Number(undefined) === NaN`
* `Number('') === 0`
* `Number(null) === 0`

```suggestion
   const score = Number(current ?? null) + 1
```